index.js

```diff
@@ -66,7 +66,19 @@ const doc = [
  * @typedef {Object.<string, number>} FileSizeMap
  */
 
+const ESCAPED_QUOTE_REGEX = new RegExp('\\\'', 'g');
```

Comment:
More concise as:

```js
const ESCAPED_QUOTE_REGEX = /\\'/g
```

Reply:
Yeah, but this doesn't work `\'` isn't removed